package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "native-template",
-  "version": "7.0.1",
+  "version": "7.0.2",
```

Comment:
This was not released yet, so you can undo this change

Reply:
done